TrafficCapture/build.gradle

```diff
@@ -10,6 +11,20 @@ allprojects {
 }
 
 subprojects {
+    ext {
```

Comment:
Instead of passing around variables, can we instead define our dependencies as a subproject reference it with platform?

i.e.

```
# TrafficCapture/trafficcapture-dependencies/build.gradle
ext {
        versions = [
            netty: '4.0.1.Final',
            jackson: '2.15.2'
        ]
}
dependencies {
  api platform("io.netty:netty-bom:${versions.netty}")

  constraints {
    // Override transitive dependency for CVE management
    api("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
  }
}
```

```
# TrafficCapture/build.gradle
subprojects {
dependencies {
  api platform(project(":trafficcapture-dependencies"))
}
```


A public example of using a similar style can be found in the [servicetalk project](https://github.com/apple/servicetalk/blob/afa2afca34eca40e05aaab0d4ee22767240bf762/servicetalk-dependencies/build.gradle)